src/main/java/dev/sigstore/json/GsonSupplier.java

```diff
@@ -30,6 +32,8 @@ public class GsonSupplier implements Supplier<Gson> {
   public Gson get() {
```

Comment:
maybe we should just singleton this. I thought we might be injecting this so I made it instantiable(?), but it's not clear we'll ever do that. But this is probably for another PR.

Reply:
#42